### DISCUSSION TITLE
[Feature Request] Binance Staking

### DISCUSSION THREAD

**Discussion by loonylabs-dev** [rotki/rotki#4151] [Archive]
Hi all,
I would really appreciate if you could implement Binance Staking.

Currently Rotki is either able show staked amounts nor show daily interested from this staking subscriptions/products

I give you some reasons why this would be a game changer and with some promotion and time you can become the must have tool for this:
* there are good staking products @Binance but most Tax/Tracking Tools costs too much per transaction and small daily staking rewards ( <1 Cent) then are not  worth it
* staking is the future for hodler; and we know that there are many investors/hodlers and only a few traders
* there is no competition out there (against rotki) that offers unlimited transactions for this affordable price
* sure Binance is at some points (especially for DEV) not a good exchange; but is has good tokens, good products, good fees and with that a big userbase

I see that there is in the GUI the function to see all the staked balances and the daily rewards/interests

Maybe you can find those rewards in /myTrades

As I was not able to run this Restful API as the signer seems to be not working (even with the imported Binance Postman collection...) so I was not able to see if those values would be returned.

**Comment by LefterisJP** [comment]
feature request is here: https://github.com/rotki/rotki/issues/2788
